### ISSUE TITLE
Firestore 0.1.0 with Kotlin Serialization 0.20.0

### ISSUE THREAD

**Issue by vanniktech** [GitLiveApp/firebase-kotlin-sdk#27]
I think (haven't checked) that firestore 0.1.0 was published with a Kotlin Serialization version other than 0.20.0.

At runtime, I'm getting:

```
No static method encode$default(Lkotlinx/serialization/SerializationStrategy;Ljava/lang/Object;Ljava/lang/Object;ILjava/lang/Object;)Ljava/lang/Object; in class Ldev/gitlive/firebase/EncodersKt; or its super classes (declaration of 'dev.gitlive.firebase.EncodersKt' appears in /data/app/app.wando.android.dev-_3L83lGJfsPdfEzZEKNkfw==/base.apk!classes3.dex)
```

Can we get a new version using 0.20.0?

**Comment by nbransby** [comment]
That looks like it can't find the encode method in the firebase-common module

**Comment by vanniktech** [comment]
Can we get #25 fixed as well?

**Comment by vanniktech** [comment]
Yeah my assumption is right. Published firebase-firestore locally with 0.20.0 and everything works now!
